cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/ClusterPartitionState.java

```diff
@@ -83,6 +86,23 @@ public Map<String, Object> getJsonStructure() {
     }
     return jsonMap;
   }
+  
+  /**
+   * Check the Kafka Partition State if it is Under Replicated or not.
+   * 
+   * @param partitionInfo the {@link PartitionInfo} object that contains partition info.
+   * @return the effective boolean under replicated status of a partition.
+   */
+  protected boolean isURP(PartitionInfo partitionInfo) {
+    int numInsyncReplicas = partitionInfo.inSyncReplicas().length;
+    Node[] inSyncReplicas = Arrays.copyOf(partitionInfo.inSyncReplicas(), numInsyncReplicas);
+    Arrays.sort(inSyncReplicas);
+    int numReplicas = partitionInfo.replicas().length;
+    Node[] replicas = Arrays.copyOf(partitionInfo.replicas(), numReplicas);
+    Arrays.sort(replicas);
+    boolean isURP = !(Arrays.equals(replicas, inSyncReplicas));
```

Comment:
Redundant brackets here. "!Arrays.equals(replicas, inSyncReplicas);" should be good enough.